src/Traveler.js

```diff
@@ -1,9 +1,53 @@
+import moment from 'moment';
+
 class Traveler {
-  constructor() {
-    this.id = id;
-    this.name = name;
-    this.travelerType = travelerType;
+  constructor(traveler) {
+    this.id = traveler.id;
+    this.name = traveler.name;
+    this.travelerType = traveler.travelerType;
+    this.allTrips = [];
+    this.pastTrips = [];
+    this.currentTrip = [];
+    this.upcomingTrips = [];
+    this.pendingTrips = [];
```

Comment:
Just a thought to a different approach here. I like to keep my classes pretty minimal, and try to avoid storing data in places that are no longer regarding the `Traveler` itself.

I haven't even built this myself, so I don't know how it would actually work out, but i'm more inclined to have a `Trips` class, and store tripStatus as a property on that class, or have arrays like line 8-12 live there. 

I don't necessarily think either approach would be 'better' than the other, so by all means, keep it this way. If you do want to explore that avenue, perhaps at a later point, let me know and I can go into further detail, or perhaps share how I have set it up.

Reply:
It actually looks like you are building a Trip class below, so we are on the same page it seems :)

Reply:
Thank you so much for your feedback Blake! I enjoy hearing all of your insights. Planning to build out my Trip class today so it's nice to know I'm heading in a good direction.